frontend/src/metabase/visualizations/components/settings/ChartSettingFieldsPicker.jsx

```diff
@@ -30,6 +30,8 @@ const ChartSettingFieldsPicker = ({
     );
   };
 
+  const isDragDisabled = Array.isArray(fields) && fields.length <= 1;
```

Comment:
I believe `fields.length === 1` will work as well, if length is 0, `map` will render nothing